src/component/Layout.js

```diff
@@ -171,7 +173,7 @@ export default ({ children, section, className, pageTitle, breadcrumbs = [] }) =
             </small>
 
             <small className="copyright">
-              Copyright &copy; 2019 <a href="https://wazo.io">Wazo</a>
+              Copyright &copy; { now.getFullYear() } <a href="https://wazo.io">Wazo</a>
```

Comment:
I think it shoud be 2019-{ now.getFullYear() }

Reply:
Both ways are right. Showing the current year only seems a bit more popular on the web.

I have no objection to add a start year. I think the right way to do it, it's to add the company created year and current year so something like `2018-YYYY`.

Which one do you prefer 😃 ?

Reply:
We cannot use 2016 as Wazo Platform didn't exist at this time. What is copyrighted has started in 2019 for me. @sboily what do you think?

Reply:
The link is wazo.io so the company. If we use wazo platform ok, but the company is started in 2016. But in general i'm not sure it's really a problem, we can use only 2020 and change every year, for me it's a detail, it doesn't matter.